config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -0,0 +1,81 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-storage-kind-disruptive
+    always_run: false
+    optional: true
+    # All the files owned by sig-storage. Keep in sync across
+    # all sig-storage-jobs
+    #
+    # pkg/controller/volume
+    # pkg/kubelet/volumemanager
+    # pkg/volume
+    # pkg/util/mount
+    # test/e2e/storage
+    # test/e2e/testing-manifests/storage-csi
+    run_if_changed: '^(pkg\/controller\/volume|pkg\/kubelet\/volumemanager|pkg\/volume|pkg\/util\/mount|test\/e2e\/storage|test\/e2e\/testing-manifests\/storage-csi)'
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    annotations:
+      testgrid-create-test-group: 'true'
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:v20230421-ec4335b54b-1.27
+        command:
+        - wrapper.sh
+        args:
+        - /bin/sh
+        - -xc
+        - >
+          make WHAT="github.com/onsi/ginkgo/v2/ginkgo k8s.io/kubernetes/test/e2e/e2e.test k8s.io/kubernetes/cmd/kubectl" &&
+          curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" kind &&
```

Comment:
Do you want to run with the latest nightly build of kind (= `kind.sigs.k8s.io/dl/latest`) or a fixed version of kind (= https://github.com/kubernetes-sigs/kind/releases)?

I'm about to switch to a fixed version in https://github.com/kubernetes/test-infra/pull/29570 because of version drift between the base image and the kind version.

This isn't an issue here because you use the base image that was built for the latest nightly, but perhaps the latest kind is too unstable?

Reply:
I can see everyone uses `latest` in their CI jobs and I'd prefer to be with the majority - if that gets broken, someone  (hopefully) updates add kinds

Reply:
Yeah, generally we'd jump on fixing anything that came up, DRA was a bit of an edge case getting ahead on containerd and we happened to be stuck on older containerd for multiple reasons[^1]. That's been resolved and we've moved DRA back to latest as well.

[^1]:  Coupled runc version shipped with containerd and runc upgrades are currently unsafe, we uncoupled those. And containerd deprecated some config fields and we needed a migration path, that's solved